### ISSUE TITLE
Do we still need travis CI?

### ISSUE THREAD

**Issue by MrSidims** [KhronosGroup/SPIRV-LLVM-Translator#1717]
From what I see we have github actions on almost all of the release branches, so is there a reason to keep travis running?

**Comment by svenvh** [comment]
Most of the newer release branches have GitHub Actions already indeed, so I'm okay with disabling Travis there.  For older release branches (before `llvm_release_100` it seems), we could keep Travis or migrate to GitHub; I don't have a strong preference.

For reference, #918 disabled Travis on the main (former master) branch.

**Comment by svenvh** [comment]
> it is highly likely that we haven't published any binaries with our releases for a while and with that in mind, we may decide to completely get rid of Travis CI.

That's correct, I don't think we have published binaries for quite a while (and nobody seems to have asked for them).  The last time we did was for release 12 according to the [releases](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/releases) page.
